app/src/main/java/xyz/whereuat/whereuat/db/command/DeleteCommand.java

```diff
@@ -34,7 +34,7 @@ public DeleteCommand(Context context, String table,
      * @return Integer representing the number of rows that were deleted from the table
      */
     @Override
-    public Integer execute() {
+    public Object call() {
```

Comment:
I actually think this return type can be `Long` because long is a subclass of `Object`. It's called a covariant return type.
http://stackoverflow.com/questions/14694852/can-overridden-methods-differ-in-return-type